main.tf

```diff
@@ -28,19 +32,15 @@ resource "aws_subnet" "public" {
     create_before_destroy = true
   }
 
-  tags {
-    Name = "${var.name}.${element(var.azs, count.index)}"
-  }
+  tags = "${merge(var.tags, map("Name", format("%s-subnet-public-%s", var.name, element(var.azs, count.index))))}"
 }
 
 # Routes
 resource "aws_route_table" "public" {
   vpc_id = "${var.vpc_id}"
   count  = "${length(var.cidrs)}"
 
-  tags {
-    Name = "${var.name}.${element(var.azs, count.index)}"
-  }
+  tags = "${merge(var.tags, map("Name", format("%s-rt-public-%s", var.name, element(var.azs, count.index))))}"
```

Comment:
I'm just wondering why it can't be change? let's say we want to keep some kind of consistency between all the modules like how resources are being named in [tf_aws_vpc](https://github.com/terraform-community-modules/tf_aws_vpc/blob/master/main.tf#L53)